pandapower/shortcircuit/currents.py

```diff
@@ -115,6 +115,8 @@ def _calc_branch_currents(net, ppc):
         ikss_all_f = abs(ikss1_all_f)
         ikss_all_t = abs(ikss1_all_t)
 
+    ikss_all_f[ikss_all_f < 1e-10] = np.nan
+    ikss_all_t[ikss_all_t < 1e-10] = np.nan

```

Comment:
Why was it changed to zero in line 99/100, but set to nan in line 118/119? Is this a different use case?

Reply:
This is a bit tricky, at line 99/100 I set to 0 because nan values in `ikss1_all_f`, `ikss1_all_t` would otherwise cancel  out any current source contribution in `ikss2_all_f`, `ikss2_all_t` at  line 112/113. Since nan + x = nan. 
Then at line 118/119 you have to put the nan back (but after the sum) to avoid taking 0 values when you take the minimum current at line 120/121, when `case='min`. 
I was actually a little dubious on this last part, I mean, theoretically taking the 0 would be the expected output from a mathematical point of view but I saw this wasn't the case in the previous version of the code so I left it as it was (as it would break all the tests).

Reply:
Ok makes sense, thanks for the explanation. It is a little dubios, and if you come up with a cleaner solution that would be great, but this will do for now :)